shared/modules/currency-display.utils.ts

```diff
@@ -1,9 +1,9 @@
 export const showPrimaryCurrency = (
   isOriginalNativeSymbol: boolean,
-  useNativeCurrencyAsPrimaryCurrency: boolean,
+  showNativeTokenAsMainBalance: boolean,
```

Comment:
I'm confused whether this file makes sense anymore now that the primary/secondary toggle is removed.  Seems like these functions are only used in the asset-list.  But I don't expect that to be affected by `showNativeTokenAsMainBalance` so maybe a bunch of this can be simplified / removed?

Reply:
I agree there is room to refactor this logic, will remove this file and move logic to assetList 🙏  [8dda377](https://github.com/MetaMask/metamask-extension/pull/27097/commits/8dda3779d0298b6896844b2dd9ed59c3b753329f)